docs/extensibility/checkout-flow-and-events.md

```diff
@@ -473,11 +476,12 @@ This event emitter is fired when a shipping rate selection is not being persiste
 
 This event emitter doesn't care about any registered observer response and will simply execute all registered observers passing them the current error status in the context.
 
-<!-- FEEDBACK -->
+ <!-- FEEDBACK -->
```

Comment:
There was some weird prettier thing going on here where it just added a couple of `##` next to the feedback for some reason. Good spot, I've corrected it here and the other docs file